src/Umbraco.Web.UI.Client/src/less/canvas-designer.less

```
@@ -14,728 +15,194 @@
 /****************************/
 
 body {
+    position: absolute;
     overflow: hidden;
+    display: flex;
+    align-items: center;
+    justify-content: center;
     height: 100%;
-    width: 100%;
-    width: calc(~"100% - 80px"); // 80px is the fixed left menu for toggling the different browser sizes
-    position: absolute;
+    height: calc(~"100% - 40px");
+    width: 100%; // 80px is the fixed left menu for toggling the different browser sizes
```

Can we remove the comment as makes no sense anymore...
```suggestion
    width: 100%;
```